src/daemons/MetaDaemon.cpp

```diff
@@ -105,6 +101,11 @@ int main(int argc, char* argv[]) {
   }
 #endif
 
+  if (FLAGS_data_path.empty()) {
```

Comment:
Maybe give a check on standalone daemon.

Reply:
The standalone daemon has been checked.